packages/wallets/src/components/AddMoreWalletsCarousel/AddMoreWalletsCarousel.scss

```
@@ -0,0 +1,79 @@
+.add-wallets {
```

```suggestion
.wallets-add-more {
```

or more appropriate name 😄

---

ok